### DISCUSSION TITLE
Progress in downloaded bytes

### DISCUSSION THREAD

**Discussion by edipo-santos** [Tyrrrz/YoutubeExplode#836] [Q&A]
Hello!

First of all, I would like to thank you for the tool; it has helped me a lot.

Is it possible to track the progress in bytes downloaded and total bytes? Or any other way I can display the progress and transfer rate in detail.

Best regards

**Comment by Tyrrrz** [comment]
Methods that except `IProgress<double>` report progress in a `0..1` range
